test/ParseableInterface/ModuleCache/SystemDependencies.swiftinterface

```diff
@@ -43,11 +43,12 @@
 // NEGATIVE-NOT: SomeCModule.h
 // CHECK: SomeCModule.h
 
-// CHECK-DUMP-NOT: usr/include
-// CHECK-DUMP: DEPENDENCY_DIRECTORY{{.+}}'usr/include'
+// CHECK-DUMP-NOT: usr{{[/\\]}}include
+// CHECK-DUMP: DEPENDENCY_DIRECTORY{{.+}}'usr{{[/\\]}}include'
 // CHECK-DUMP-NEXT: FILE_DEPENDENCY{{.+}}'module.modulemap'
-// CHECK-DUMP-NEXT: FILE_DEPENDENCY{{.+}}'SomeCModule.h'
-// CHECK-DUMP-NOT: usr/include
+// CHECK-DUMP-NOT: FILE_DEPENDENCY{{.+}}'SomeCModule.h'
```

Comment:
Sorry, I meant `CHECK-DUMP-NOT: usr{{[/\\]}}include`. The point is that we don't want to serialize the directory part twice.